app/models/user_resource.rb

```diff
@@ -2,7 +2,7 @@
 
 class UserResource < ApplicationRecord
   has_and_belongs_to_many :tags
-  has_one_attached :file, service: :microsoft
+  has_one_attached :file, service: :local
```

Comment:
das muss auf :microsoft bleiben

Reply:
Ihr solltet sowas eigentlich nicht auf Model-Ebene einstellen. Das ist zwar möglich, aber nicht dafür gedacht. Dies ist für "Sonderfälle" vorbehalten.

Die richtige Config gehört in `config/storage.yml`
https://edgeguides.rubyonrails.org/active_storage_overview.html#setup

Ihr solltet für Tests und Dev nicht immer MS mit Daten befeuern.